doc/book/in-depth-guide/zend-db-sql-zend-hydrator.md

```diff
@@ -293,22 +293,19 @@ public function findAllPosts()
 Refreshing the application you should now see output similar to the following:
 
 ```text
-Zend\Db\Adapter\Driver\Pdo\Result::__set_state(array(
```

Comment:
I'm reverting this one during merge, as this is the result of running var_dump, which *never* uses short array notation. We want this to look the same as what PHP spits out.